code/game/objects/effects/decals/crayon.dm

```diff
@@ -697,54 +866,58 @@ obj/item/scroll/attackby(obj/item/I, mob/living/carbon/human/M)
 			M.maxHealth += 1
 			M.health += 1
 			M.unnatural_mutations.total_instability += 1 //A soft cap
-		B.remove_self(70)
-		lesser.dust()
-		M.sanity.changeLevel(-20)
+			B.remove_self(70)
+			lesser.dust()
+			M.sanity.changeLevel(-10)
 		return
 	return
 
+// Cards: Invokes a random carp card, to use with other spells
 /obj/effect/decal/cleanable/crayon/proc/cards_spell(mob/living/carbon/human/M)
 	var/datum/reagent/organic/blood/B = M.get_blood()
-	to_chat(M, "<span class='warning'>A voice whispers infront you. Any foils?</span>")
+	to_chat(M, "<span class='warning'>A voice whispers in front of you: Any foils?</span>")
 	for(var/obj/item/device/camera_film in oview(1)) // Must be on the spell circle
 		B.remove_self(15)
-		new /obj/random/card_carp(src.loc)
+		new /obj/item/pack_card_carp(src.loc) // For its blood price this feels more fair.
 		M.sanity.changeLevel(-3)
 	return
 
+// Cards to Life: Consumes a Carp card of a certain type to summon an animal accordingly.
+// The animal in question is tamed, friendly to the colony, but are incredibly frail and weak.
+// Pelt cards can be turned into scroll pouches, Warren turns into a burrow
 /obj/effect/decal/cleanable/crayon/proc/cards_to_life_spell(mob/living/carbon/human/M)
 	var/datum/reagent/organic/blood/B = M.get_blood()
 	var /mob/living/simple_animal/simplemob = /mob/living/simple_animal/hostile/creature
 	var /mob/living/carbon/superior_animal/superiormob = null
 	for(var/obj/item/card_carp/carpy in oview(1))
-		M.sanity.changeLevel(1)
-		to_chat(M, "<span class='warning'>The card rotates 90 degrees then begins to fold, twisting till it breaks open with a ripping sound.</span>")
-		B.remove_self(50)
```

Comment:
Blood cost 100% needed and keep the sanity restoring

Reply:
I moved the blodcost and sanity changes down below where the things spawn, be it animals or the pouch or the burrow